urbanopt_to_alfalfa.ipynb

```diff
@@ -0,0 +1,293 @@
+    "### Define Alfalfa simulation parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "414e3ec4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you are using historian, you will need to search for this time period in Grafana dashboard to view results.\n",
+    "start_dt = datetime.datetime(2021, 7, 1, 12, 2, 0)\n",
+    "end_dt = datetime.datetime(2021, 7, 3, 0, 0, 0)\n",
+    "\n",
+    "# For external_clock == true, API calls are used to advance the model.  \n",
+    "# If external_clock == false, Alfalfa will handle advancing the model according to a specified timescale (timescale 1 => realtime)\n",
+    "params = {\n",
+    "    \"external_clock\": True,\n",
+    "    \"start_datetime\": start_dt,\n",
+    "    \"end_datetime\": end_dt\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f10a562c",
+   "metadata": {},
+   "source": [
+    "### Create zipped folders to upload to Alfalfa from the URBANopt Project\n",
+    "\n",
+    "The code below will create zipped folders for each URBANopt building model that can be uploaded to Alfalfa. The Zipped folder contains: \n",
+    "\n",
+    "    - Model Folder: Contains OpenStudio model (.osm file) for each building created using the URBANopt simulation.\n",
+    "    - Measures Folder: Measures to be added while running Alfalfa in the OpenStudio Workflow\n",
+    "    - Weather Folder: Contains EPW weather file \n",
+    "    - workflow.osw file: OpenStudio workflow file\n",
+    "\n",
+    "**Define the following variables before running the code**:\n",
+    "\n",
+    "- `uo_folder` : URBANopt Scenario Directory \n",
+    "- `weather`: Define weather file used in URBANopt project\n",
+    "- `workflow`: OpenStudio workflow file name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83836f8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set URBANopt scenario directory\n",
+    "uo_folder = 'C:/a/baseline_stochastic/run/baseline_scenario_stochastic'\n",
+    "\n",
+    "uo_scenario_folder = uo_folder.split(\"/\")[-1]\n",
+    "uo_scenario_path = os.path.join('./{}'.format(uo_scenario_folder))\n",
+    "Path(uo_scenario_path).mkdir(parents=True, exist_ok=True)\n",
+    "\n",
+    "print(\"Zipped folders saved at {}\".format(uo_scenario_path))\n",
+    "\n",
+    "# Add .epw filename\n",
+    "weather = \"USA_CO_Denver.Intl.AP.725650_TMY3.epw\"\n",
+    "\n",
+    "# Add .osw filename\n",
+    "workflow = \"workflow.osw\"\n",
+    "\n",
+    "for filename in os.listdir(uo_folder):\n",
+    "        \n",
+    "    if os.path.isfile(os.path.join(uo_folder, filename, 'in.osm').replace(\"\\\\\",\"/\")):\n",
+    "        uo_model_filepath = os.path.join(uo_scenario_path, filename)\n",
+    "        Path(uo_model_filepath).mkdir(parents=True, exist_ok=True)\n",
+    "        Path(os.path.join(uo_model_filepath, 'models')).mkdir(parents=True, exist_ok=True)\n",
```

Comment:
if `uo_model_filepath` was already of type `Path` this could look like 
```python
(uo_model_filepath / 'models').mkdir(parents = True, exists_ok = True)
```